rancher/manifest.yaml

```diff
@@ -0,0 +1,5 @@
+name: Rancher
+version: stable
+maintainer: "@saiyampathak"
+description: "Rancher - Run Kubernetes Everywhere, from datacenter to cloud to edge, Rancher lets you deliver Kubernetes-as-a-Service."
```

Comment:
This is a bit just "salesy", could you maybe use:

> Addresses the operational and security challenges of managing multiple Kubernetes clusters with integrated tools for running containerized workloads

It's a bit more descriptive of what it does (while still taken from Rancher.com)

Reply:
That shouldn't, the lint check is 160 characters and the text I pasted is 149 characters.

Reply:
Sorry didn't see your early version - I just looked at the latest changes. So yes, your early version was too long, the second one didn't really fit with the other applications (a description for our users to explain what it does, without being too salesy), so the version I edited down from theirs may be best :-)